### ISSUE TITLE
Release v0.1.0

### ISSUE THREAD

**Issue by ynamiki** [hyperledger-labs/minbft#172]
Discuss what are included and when it will be released here.

**Comment by sergefdrv** [comment]
From my side, I would like to at least add support for view change in this release. It would also be nice to add some description of implementation design and required protocol adjustments.

**Comment by ynamiki** [comment]
@sergefdrv Thanks, can I have a rough idea of your schedule?

**Comment by ynamiki** [comment]
I should update the CI environment: the latest Go, SGX SDK, linter, and OS?, etc. If the latest SGX SDK supports "containerization", I will do that (#163).

**Comment by ynamiki** [comment]
@Naoya-Horiguchi Do you have items for this release?

**Comment by nhoriguchi** [comment]
> @Naoya-Horiguchi Do you have items for this release?

I'll work on testing framework (#137) in this quarter, so please add it to the item list.

**Comment by sergefdrv** [comment]
> @sergefdrv Thanks, can I have a rough idea of your schedule?

@ynamiki Sorry for late reply. I am thinking to finish this in about couple of months.

**Comment by ynamiki** [comment]
Thanks, I have set the due date to the end of July. This is not a strict deadline; we will update it depending on our progress.

https://github.com/hyperledger-labs/minbft/milestone/1

@sergefdrv @Naoya-Horiguchi Please create issues and set the project ("Development") and a milestone if you have a task that is not presented as an issue.

**Comment by ynamiki** [comment]
@sergefdrv @Naoya-Horiguchi I have set the due date of the milestone [v0.1.0](https://github.com/hyperledger-labs/minbft/milestone/1) to the end of this month, but I think it needs some more time to finish the issues under the milestone. I would like to have your idea about schedule of your issues.

**Comment by nhoriguchi** [comment]
> @sergefdrv @Naoya-Horiguchi I have set the due date of the milestone [v0.1.0](https://github.com/hyperledger-labs/minbft/milestone/1) to the end of this month, but I think it needs to some more time to finish the issues under the milestone. I would like to have your idea about schedule of your issues.

I agree that we need more time for v0.1. Unfortunately I'm busy on other project now, so no visible update on my item. Maybe adjusting the target date to the end of July seems fine to me.

**Comment by sergefdrv** [comment]
@ynamiki Sorry, I forgot to tell in advance, but I am also quite busy on some other projects. I think, in the worst case, it could take me up to the end of October to finish preparing all pieces of view change implementation for review. I don't mind tentatively setting the target date more optimistically, like the end of July.

**Comment by ynamiki** [comment]
@sergefdrv @Naoya-Horiguchi Thanks, the due date is updated to the end of July.

**Comment by ynamiki** [comment]
@sergefdrv @Naoya-Horiguchi I think we need to update the target date again. From the previous comment of @sergefdrv , the end of November (the codes are prepared at the end of October and another month to review, etc.) is my idea. Could I have your ideas?

**Comment by nhoriguchi** [comment]
@ynamiki I agree with you, end of November looks more realistic schedule to me.

**Comment by sergefdrv** [comment]
@ynamiki The new target date seems realistic to me. We can always release it earlier, if we manage to :wink:

**Comment by ynamiki** [comment]
Thanks, target is now the end of November.
https://github.com/hyperledger-labs/minbft/milestone/1

**Comment by ynamiki** [comment]
@sergefdrv @Naoya-Horiguchi I'm afraid we need more time for this release. Could I have your plan of your development items?

**Comment by sergefdrv** [comment]
@ynamiki I hope I can come back to this and continue preparing view change implementation in January. I suppose end of February would be a fairly safe assumption to deliver it. But I will try to make it faster.

**Comment by ynamiki** [comment]
Thanks, now the due date of v0.1 is the end of February.
https://github.com/hyperledger-labs/minbft/milestone/1

**Comment by ynamiki** [comment]
@sergefdrv @Naoya-Horiguchi Should we postpone the release, which is planed at the end of this month?

**Comment by sergefdrv** [comment]
Unfortunately, yes, I am still too busy with other tasks...

**Comment by ynamiki** [comment]
Sure, I deleted the due date from the milestone.

**Comment by sergefdrv** [comment]
@ynamiki I have opened all pull requests that I planed for this release
